core/domain/learner_group_services.py

```diff
@@ -494,6 +506,50 @@ def add_student_to_learner_group(
     learner_group_model.put()
 
 
+def remove_students_from_learner_group(
+    group_id: str,
+    user_ids: List[str],
+    update_group: bool
```

Comment:
Why this additional flag? What is a case where you need to only update the usermodel but not the learnergroupmodel?